packages/lib/translations/ro/marketing.po

```diff
@@ -0,0 +1,594 @@
+msgid "Amount Raised"
+msgstr "Suma Strânsă"
+
+#: apps/marketing/src/components/(marketing)/pricing-table.tsx:145
+#: apps/marketing/src/components/(marketing)/pricing-table.tsx:189
+msgid "API Access"
+msgstr "Acces API"
+
+#: apps/marketing/src/components/(marketing)/faster-smarter-beautiful-bento.tsx:63
+msgid "Beautiful."
+msgstr "Frumos."
+
+#: apps/marketing/src/components/(marketing)/faster-smarter-beautiful-bento.tsx:65
+msgid "Because signing should be celebrated. That’s why we care about the smallest detail in our product."
+msgstr "Pentru că semnarea trebuie să fie sărbătorită. De aceea ne pasă de cel mai mic detaliu din produsul nostru."
+
+#: apps/marketing/src/components/(marketing)/footer.tsx:35
+#: apps/marketing/src/components/(marketing)/header.tsx:57
+#: apps/marketing/src/components/(marketing)/mobile-navigation.tsx:36
+msgid "Blog"
+msgstr "Blog"
+
+#: apps/marketing/src/components/(marketing)/open-build-template-bento.tsx:60
+msgid "Build on top."
+msgstr "Construiește pe baza."
+
+#: apps/marketing/src/app/(marketing)/pricing/page.tsx:163
+msgid "Can I use Documenso commercially?"
+msgstr "Pot folosi Documenso comercial?"
+
+#: apps/marketing/src/components/(marketing)/footer.tsx:42
+msgid "Careers"
+msgstr "Cariere"
+
+#: apps/marketing/src/components/(marketing)/footer.tsx:36
+msgid "Changelog"
+msgstr "Istoric de Modificări"
+
+#: apps/marketing/src/components/(marketing)/open-build-template-bento.tsx:81
+msgid "Choose a template from the community app store. Or submit your own template for others to use."
+msgstr "Alegeți un șablon din magazinul de aplicații al comunității. Sau trimiteți propriul șablon pentru ca alții să îl folosească."
+
+#: apps/marketing/src/app/(marketing)/open/page.tsx:219
+msgid "Community"
+msgstr "Comunitate"
+
+#: apps/marketing/src/app/(marketing)/open/monthly-completed-documents-chart.tsx:55
+msgid "Completed Documents"
+msgstr "Documente Finalizate"
+
+#: apps/marketing/src/app/(marketing)/open/monthly-completed-documents-chart.tsx:33
+msgid "Completed Documents per Month"
+msgstr "Documente Finalizate pe Lună"
+
+#: apps/marketing/src/components/(marketing)/share-connect-paid-widget-bento.tsx:61
+msgid "Connections"
+msgstr "Conexiuni"
+
+#: apps/marketing/src/components/(marketing)/enterprise.tsx:35
+msgid "Contact Us"
+msgstr "Contactați-ne"
+
+#: apps/marketing/src/components/(marketing)/share-connect-paid-widget-bento.tsx:63
+msgid "Create connections and automations with Zapier and more to integrate with your favorite tools."
+msgstr "Creați conexiuni și automatizări cu Zapier și altele pentru a integra cu instrumentele dvs. preferate."
+
+#: apps/marketing/src/components/(marketing)/call-to-action.tsx:23
+msgid "Create your account and start using state-of-the-art document signing. Open and beautiful signing is within your grasp."
+msgstr "Creați-vă contul și începeți să utilizați semnarea documentelor de ultimă generație. Semnarea deschisă și frumoasă este la îndemâna dvs."
```

Comment:
`"Creați-vă un cont și începeți să utilizați Documenso pentru semnaturi digital. Experimentați simplicitatea și rapiditatea semnăturii electronice."`

Something like this. It's really difficult to translate it 1-to-1 and actually sound good.